### ISSUE TITLE
Not reproducible result

### ISSUE THREAD

**Issue by vogoriachko** [YBZh/MultiClassDA#1]
Hello.
I am trying to reproduce your result from paper for Office-31 dataset.
You do not provide the correct configuration for Office-31 for the best result in the paper (in repo for SymmNet only one strange version with 11 class were found). So I follow the default parameters you have provided with 400 epoch and 31 classes for Office-31. 
I've got the following results 
W -> A accuracy: 73.9 instead the best 77.0
A -> D accuracy: 88.15 instead the best 95.6
So I can not get the same result as you even for SymmNets-V2. Can you provide the appropriate parameters to get the same score? 
Also in code you mentioned your best model 
from solver.SymmNetsV2SC_solver import SymmNetsV2SolverSC as Solver
But there is not code for solver.SymmNetsV2SC_solver. Are you going to release it ?

**Comment by YBZh** [comment]
Sorry for the delayed reply. You can contact me via e-mail for a fast response. 

It seems that you adopt the configuration of open set UDA (11 classes) for the use of closed UDA (31 classes). Their codes are different in the dataloader as well as the criterion of final results. I will make an update this week, including the clarification of configuration for different tasks settings and the example results on the W->A and A->D.

**Comment by YBZh** [comment]
There are so many experiments associated with this project, so users may get confused on how to conduct experiments.

Thanks for pointing out this problem.  We have provided an clarification on the experiments in the file of 'https://github.com/YBZh/MultiClassDA/blob/master/experiments/Clarification_on_tasks.md' 
Additionally, we provide the results of W->A and A->D in the file of 'https://github.com/YBZh/MultiClassDA/tree/master/experiments/SymmNets'  and you can obtain these results with the script in the 'https://github.com/YBZh/MultiClassDA/blob/master/run_temp.sh'. 

We observe improved results with a new edition of PyTorch:
| Tasks  | SymmNets-V2 (in paper) |  SymmNets-V2 (this code) | SymmNets-V2-SC (in paper) |SymmNets-V2-SC (this code) |
| ------------- | ------------- |  ------------- | ------------- |  ------------- | 
| W2A  | 73.4  |  74.0 (max) | 77.0 |  77.6 |
| A2D  | 93.5   |  94.8 | 95.6 |  95.4 (max) |
